src/Dom/Document.php

```diff
@@ -536,6 +537,35 @@ private function move_invalid_head_nodes_to_body() {
 		}
 	}
 
+	/**
+	 * Converts a possible head[profile] attribute to link[rel=profile].
+	 *
+	 * The head[profile] attribute is only valid in HTML4, not HTML5.
+	 * So if it exists and there's no link[rel=profile], add it to the <head> as a link[rel=profile].
+	 */
+	private function convert_head_profile_to_link() {
+		$profile = $this->head->getAttribute( 'profile' );
+		if ( ! $profile ) {
+			return;
+		}
+
+		// If there's already a head > link[rel=profile], don't create another.
+		$links = $this->head->getElementsByTagName( 'link' );
+		foreach ( $links as $link ) {
+			if ( 'profile' === $link->getAttribute( 'rel' ) ) {
+				$this->head->removeAttribute( 'profile' );
+				return;
+			}
+		}
```

Comment:
I don't think this is necessary. It's not a problem to have another, is it?

Reply:
Yeah, good point. According to the [w3.org validator](https://validator.w3.org/), it's still valid HTML if there are 2 `link[rel=profile]` tags:

![valid-html](https://user-images.githubusercontent.com/4063887/73397509-4e5f5c00-42a9-11ea-8901-44d2bcd385c5.png)

Reply:
Reverted in https://github.com/ampproject/amp-wp/pull/4193/commits/2294c6e9412683aba413ca9e1286776add32b9e6